shared/search/user-pane/index.js

```diff
@@ -49,7 +49,7 @@ export default connector.connect(
       return {
         mode: 'external',
         nonUserInfoProps: {
-          avatar: userForInfoPane.serviceAvatar,
+          avatar: userForInfoPane.extraInfo.serviceAvatar || '',
```

Comment:
why this change?

you aren't sure extraInfo will be the variant that contains the serviceAvatar